File: Pidev_EGOV/piegovj2me/src/edu/esprit/piegov/entites/CarteGrise.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package edu.esprit.piegov.entites;

import java.util.Date;

/**
 *
 * @author boumaiza
 */
public class CarteGrise {
   
    int id_carte_grise;
    String marque_vehicule;
    String num_serie_vehicule;
    String type_vehicule;
    Date date_achat_vehicule;
    String couleur_vehicue;
    String num_cinUtilisateur;
    String validation;
    
    

    public CarteGrise(int id_carte_grise, String marque_vehicule) {
        this.id_carte_grise = id_carte_grise;
        this.marque_vehicule = marque_vehicule;
    }

    public CarteGrise(String marque_vehicule, String validation) {
        this.marque_vehicule = marque_vehicule;
        this.validation = validation;
    }

    public CarteGrise(int id_carte_grise, String marque_vehicule, String num_serie_vehicule, String type_vehicule, Date date_achat_vehicule, String couleur_vehicue, String num_cinUtilisateur, String validation) {
        this.id_carte_grise = id_carte_grise;
        this.marque_vehicule = marque_vehicule;
        this.num_serie_vehicule = num_serie_vehicule;
        this.type_vehicule = type_vehicule;
        this.date_achat_vehicule = date_achat_vehicule;
        this.couleur_vehicue = couleur_vehicue;
        this.num_cinUtilisateur = num_cinUtilisateur;
        this.validation = validation;
    }

    public CarteGrise(int id_carte_grise, String marque_vehicule, String num_serie_vehicule, String type_vehicule, String couleur_vehicue, String validation) {
        this.id_carte_grise = id_carte_grise;
        this.marque_vehicule = marque_vehicule;
        this.num_serie_vehicule = num_serie_vehicule;
        this.type_vehicule = type_vehicule;
        this.couleur_vehicue = couleur_vehicue;
        this.validation = validation;
    }

 
    
   
    public CarteGrise() {
        //throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }

    public int getId_carte_grise() {
        return id_carte_grise;
    }

    public void setId_carte_grise(int id_carte_grise) {
        this.id_carte_grise = id_carte_grise;
    }

    public String getMarque_vehicule() {
        return marque_vehicule;
    }

    public void setMarque_vehicule(String marque_vehicule) {
        this.marque_vehicule = marque_vehicule;
    }

    public String getNum_serie_vehicule() {
        return num_serie_vehicule;
    }

    public void setNum_serie_vehicule(String num_serie_vehicule) {
        this.num_serie_vehicule = num_serie_vehicule;
    }

    public String getType_vehicule() {
        return type_vehicule;
    }

    public void setType_vehicule(String type_vehicule) {
        this.type_vehicule = type_vehicule;
    }

    public Date getDate_achat_vehicule() {
        return date_achat_vehicule;
    }

    public void setDate_achat_vehicule(Date date_achat_vehicule) {
        this.date_achat_vehicule = date_achat_vehicule;
    }

    public String getCouleur_vehicue() {
        return couleur_vehicue;
    }

    public void setCouleur_vehicue(String couleur_vehicue) {
        this.couleur_vehicue = couleur_vehicue;
    }

    public String getNum_cinUtilisateur() {
        return num_cinUtilisateur;
    }

    public void setNum_cinUtilisateur(String num_cinUtilisateur) {
        this.num_cinUtilisateur = num_cinUtilisateur;
    }

    public String getValidation() {
        return validation;
    }

    public void setValidation(String validation) {
        this.validation = validation;
    }

 
    
    
    
    
}

File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/CarteGrise.php
<?php

namespace MyApp\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * CarteGrise
 *
 * @ORM\Table(name="carte_grise")
 * @ORM\Entity
 */
class CarteGrise
{
     /**
     * @var integer
     *
     * @ORM\Column(name="id_carte_grise", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idCarteGrise;

    /**
     * @var string
     *
     * @ORM\Column(name="marque_vehicule", type="string", length=20, nullable=false)
     */
    private $marqueVehicule;

    /**
     * @var string
     *
     * @ORM\Column(name="num_serie_vehicule", type="string", length=9, nullable=false)
     */
    private $numSerieVehicule;

    /**
     * @var string
     *
     * @ORM\Column(name="type_vehicule", type="string", length=20, nullable=false)
     */
    private $typeVehicule;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_achat_vehicule", type="date", nullable=false)
     */
    private $dateAchatVehicule;

    /**
     * @var string
     *
     * @ORM\Column(name="couleur_vehicue", type="string", length=20, nullable=false)
     */
    private $couleurVehicue;
    
     /**
     * @var string
     *
     * @ORM\Column(name="validation", type="string", length=10, nullable=true)
     */
    private $validation;
    
    
    
    /**
    * @ORM\ManyToOne(targetEntity="User", inversedBy="carteGrise" )
    * @ORM\JoinColumn(name="num_cinUtilisateur", referencedColumnName="id")
    */
    private $utilisateur;


   

  

    /**
     * Get idCarteGrise
     *
     * @return integer 
     */
    public function getIdCarteGrise()
    {
        return $this->idCarteGrise;
    }

    /**
     * Set marqueVehicule
     *
     * @param string $marqueVehicule
     * @return CarteGrise
     */
    public function setMarqueVehicule($marqueVehicule)
    {
        $this->marqueVehicule = $marqueVehicule;

        return $this;
    }

    /**
     * Get marqueVehicule
     *
     * @return string 
     */
    public function getMarqueVehicule()
    {
        return $this->marqueVehicule;
    }

    /**
     * Set numSerieVehicule
     *
     * @param string $numSerieVehicule
     * @return CarteGrise
     */
    public function setNumSerieVehicule($numSerieVehicule)
    {
        $this->numSerieVehicule = $numSerieVehicule;

        return $this;
    }

    /**
     * Get numSerieVehicule
     *
     * @return string 
     */
    public function getNumSerieVehicule()
    {
        return $this->numSerieVehicule;
    }

    /**
     * Set typeVehicule
     *
     * @param string $typeVehicule
     * @return CarteGrise
     */
    public function setTypeVehicule($typeVehicule)
    {
        $this->typeVehicule = $typeVehicule;

        return $this;
    }

    /**
     * Get typeVehicule
     *
     * @return string 
     */
    public function getTypeVehicule()
    {
        return $this->typeVehicule;
    }

    /**
     * Set dateAchatVehicule
     *
     * @param \DateTime $dateAchatVehicule
     * @return CarteGrise
     */
    public function setDateAchatVehicule($dateAchatVehicule)
    {
        $this->dateAchatVehicule = $dateAchatVehicule;

        return $this;
    }

    /**
     * Get dateAchatVehicule
     *
     * @return \DateTime 
     */
    public function getDateAchatVehicule()
    {
        return $this->dateAchatVehicule;
    }

    /**
     * Set couleurVehicue
     *
     * @param string $couleurVehicue
     * @return CarteGrise
     */
    public function setCouleurVehicue($couleurVehicue)
    {
        $this->couleurVehicue = $couleurVehicue;

        return $this;
    }

    /**
     * Get couleurVehicue
     *
     * @return string 
     */
    public function getCouleurVehicue()
    {
        return $this->couleurVehicue;
    }

    /**
     * Set validation
     *
     * @param string $validation
     * @return CarteGrise
     */
    public function setValidation($validation)
    {
        $this->validation = $validation;

        return $this;
    }

    /**
     * Get validation
     *
     * @return string 
     */
    public function getValidation()
    {
        return $this->validation;
    }

    /**
     * Set utilisateur
     *
     * @param \MyApp\UserBundle\Entity\User $utilisateur
     * @return CarteGrise
     */
    public function setUtilisateur(\MyApp\UserBundle\Entity\User $utilisateur = null)
    {
        $this->utilisateur = $utilisateur;

        return $this;
    }

    /**
     * Get utilisateur
     *
     * @return \MyApp\UserBundle\Entity\User 
     */
    public function getUtilisateur()
    {
        return $this->utilisateur;
    }
}
